### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by matthgyver** [cryptoadvance/specter-diy#129]
Hi, 

When I'm trying to compile (``make disco``) from source just cloned from this repo, I got this error :
```
arm-none-eabi-ld: build-STM32F469DISC/scard/reader.o:(.rodata.scard_conn_params_type+0x0): multiple definition of `scard_conn_params_type'; build-STM32F469DISC/scard/scard.o:(.rodata.scard_conn_params_type+0x0): first defined here
arm-none-eabi-ld: build-STM32F469DISC/scard/connection.o:(.rodata.scard_conn_params_type+0x0): multiple definition of `scard_conn_params_type'; build-STM32F469DISC/scard/scard.o:(.rodata.scard_conn_params_type+0x0): first defined here
```

Have you an idea please ?

Thank's

**Comment by miketlk** [comment]
Hi,
Could you please check the version of the toolchain you are using, running something like:

```shell
arm-none-eabi-gcc --version 
```

**Comment by matthgyver** [comment]
Hi @miketlk 
My version is : ``arm-none-eabi-gcc (Arch Repository) 10.2.0``

**Comment by miketlk** [comment]
Thanks, @matthgyver
It looks like GCC 10.x has a stricter treatment of the "external" keyword (or the lack of it) for constants.
 
Fixed in 65e719a1c758167d0309411d09b663d6d25112f9
